haiku-apps/bemobile/bemobile-0.9.recipe

```diff
@@ -0,0 +1,38 @@
+SUMMARY="A program to manage contacts and messages stored on your mobile phone"
+DESCRIPTION="BeMobile is a program to manage contacts and messages stored on your mobile phone. It has been developed for Motorola C550 and L6 models but should work with any phone that supports AT command set."
+HOMEPAGE="https://github.com/ytmytm/beos-bemobile"
+COPYRIGHT="<2007 Maciej Witkowiak>"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="!"
```

Comment:
You can prepare the SOURCE_URI with the eventual commit hash. Compare the use of "srcGitRev" in e.g. armyknife-5.1.0 .

Reply:
Quick link:
https://github.com/haikuports/haikuports/blob/c2a214e6473117adfdbf84d0f792341b30e1bbd7/haiku-apps/armyknife/armyknife-5.1.0.recipe#L12

(Of course, for this recipe it will be a different hash.)